readme.md

```diff
@@ -118,27 +118,27 @@ mv readme.txt documents/
 mv readme.txt README.md
 ```
 
-## move a folder
+## move a directory
 
-**STOP DRAG AND DROPING A FOLDER, OR CMD/CTRL + X, CMD/CTRL + V A FOLDER** :-1:
+**STOP DRAG AND DROPING A directory, OR CMD/CTRL + X, CMD/CTRL + V A directory** :-1:
```

Comment:
Could you make these capital? Thanks.

Reply:
Done, also corrected 'subdirectorys' to 'subdirectories'.